src/index.html

```diff
@@ -7,7 +7,9 @@
   <body>
     <main class="sc-dcs dcs-full-page">
       <div class="dcs-full-page__container">
-        <a class="dcs-close"></a>
+        <!-- <div class="dcs-full-page__close">
```

Comment:
I didn't see the close button in the sketchfile on the full page, so I've hidden it